test/StakeManager.js

```diff
@@ -1000,5 +1193,21 @@ describe('StakeManager', function () {
       assertBNEqual(await sToken.balanceOf(signers[8].address), tokenAmount('1000'), 'Staker Balance MisMatch');
       assertBNEqual(await sToken.balanceOf(signers[9].address), toBigNumber('5').mul(BigNumber.from(10).pow(BigNumber.from(17))), 'Delegator Balance MisMatch');
     });
+
+    it('Staker should not be able to withdraw if his stake is zero', async function () {
+      let epoch = await getEpoch();
+      const stakerIdacc3 = await stakeManager.stakerIds(signers[3].address);
+      await stakeManager.connect(signers[3]).unstake(epoch, stakerIdacc3, tokenAmount('1000'));
+      for (let i = 0; i < WITHDRAW_LOCK_PERIOD - 1; i++) {
+        await mineToNextEpoch();
+      }
+      await mineToNextEpoch();
```

Comment:
why is this here? just remove -1 is the loop above.

Reply:
ok